### PR TITLE
feat(RELEASE-987): add `check-jsonschema` dependency to Dockerfile

### PR DESCRIPTION
**Pull request by seanconroy2021**
Add check-jsonschema to be used in the check-data-keys task in RSC.

Open PR that need this dependency  [PR](https://github.com/konflux-ci/release-service-catalog/pull/603)

### CODE REVIEW

**Review by mmalina — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -42,6 +42,7 @@ RUN curl -LO https://github.com/release-engineering/exodus-rsync/releases/latest
     chmod +x exodus-rsync && mv exodus-rsync /usr/local/bin/rsync
 
 RUN pip3 install jinja2 \
+    check-jsonschema \
```
[the remark this answers is not in the document]
Comment (seanconroy2021) [reply Dockerfile:45]: Yea, I started off with [jsonschema](https://github.com/python-jsonschema/jsonschema), but that is depreciated.

`DeprecationWarning: The jsonschema CLI is deprecated and will be removed in a future version. Please use check-jsonschema instead, which can be installed from https://pypi.org/project/check-jsonschema/`

**Review by johnbieren — APPROVED** [review]

**Pull request closed without merging**